context.go

```diff
@@ -16,14 +16,41 @@ import (
 	"github.com/sdboyer/gps"
 )
 
+type BuildContext interface {
```

Comment:
Yeah, I'm not a fan of that aspect, either. I considered plain `Context`; while it is what `go/build` does, that was also before the `context` package became a thing. Seems like it dominates that naming now, and I didn't have an alternative, sooo... 😄